src/data/formInformation.js

```diff
@@ -0,0 +1,51 @@
+export const initialContent = [
+	{
+		text: "Don't miss the 1999 Silver Jubilee event! Register now to secure your spot and confirm your registration by making a payment.",
+		classes: 'mb-2',
+	},
+	{
+		text: "We understand that plans can change. That's why we offer a full refund if you need to cancel your registration until September 30th.",
+		classes: 'py-2 ',
+	},
+];
+
+export const feeCoverage = [
+	'Cultural and Entertainment activities for Dec 22nd and Dec 23rd.',
+	'Food and non-alcoholic beverages for Dec 22nd evening through Dec 24th lunch.',
+	'Stage setup, photography, party hall rental, music systems, orchestra, etc.',
+	'Souvenirs and gifts for registered Alumni.',
+	'Teacher felicitations, and other memorabilia.',
+	'Registration kit which includes lanyard with name tag, ID card for NITR campus visit, goodie bag and event itinerary.',
+	'Transportation and logistics to and from hotel to venue/NIT Rourkela from Dec 22nd through Dec 24th.',
+];
+
+export const inputContent = [
```

Comment:
https://github.com/dscnitrourkela/project-raisin/blob/main/config/content/Registration.js

A format like this will be better if we want to manage and validate states.

Reply:
fixed. https://github.com/dscnitrourkela/project-udon/pull/11/commits/df2c60b6e78acf74bf8f1bd45130175472b40943